js/components/navigationBar.js

```diff
@@ -70,7 +70,11 @@ class NavigationBar extends ImmutableComponent {
     contextMenus.onReloadContextMenu(target)
   }
 
-  onHome () {
+  onHome (e) {
+    if (e.ctrlKey) {
```

Comment:
instead of e.ctrlKey, can you change this to use `isForSecondaryAction()` in `eventUtil`?  Thanks 😄